extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/model/TransferRequestDto.java

```diff
@@ -34,6 +34,7 @@
     private TransferType transferType;
     private String protocol = "ids-multipart";
     private String connectorId;
+    private String assetId;
```

Comment:
Just FYI: I think eventually the `DataRequest` should only carry the `contractId`, and no `assetId`, because provider connectors should be to resolve the asset through the contract. Not for this PR though, just an FYI.

Reply:
I agree, let's do that in another PR.